cost/google/cud_report/Google_Committed_UseDiscount_Report.pt

```diff
@@ -0,0 +1,134 @@
+name "Google Committed Use Discount (CUD) Report"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on all Google CUDs that have been purchased. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/google/cud_report) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
In an attempt to educate the user, recommend adding an additional line to the short description in line with the note in the readme about best practices in applying the policy.

>short_description "Report on all Google CUDs that have been purchased. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/google/cud_report) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more. \n
>As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account."

Reply:
@ryanoleary Should we specify "single account" as opposed to "master account"?